general/example_damped_numerical_filtered_controller.m

```diff
@@ -21,11 +21,13 @@ function example_damped_numerical_filtered_controller()
     xd = DQ([1,0,0,0,0,0,0,0.652495]);
 
     %% Robot Declaration
-    schunk_dh = [0,     0,   0,     0,   0,      0,  0;
-                 0.3,   0,   0.328, 0,   0.2765, 0,  0.40049;
-                 0,     0,   0,     0,   0,      0,  0;
-                -pi2,   pi2,-pi2,   pi2,-pi2,    pi2,0];
-    schunk = DQ_SerialManipulator(schunk_dh, 'standard');
+    schunk_dh = [0,     0,   0,     0,   0,      0,  0;  % Theta
+                 0.3,   0,   0.328, 0,   0.2765, 0,  0.40049; % d
+                 0,     0,   0,     0,   0,      0,  0;       % a
+                -pi2,   pi2,-pi2,   pi2,-pi2,    pi2,0;       % Alpha
+                 repmat(DQ_SerialManipulatorDH.JOINT_ROTATIONAL,1,7);  % Type of joint                                      
+                ];
```

Comment:
Please change this line to

```cpp
schunk = DQ_SerialManipulatorDH(schunk_dh);
```

Because the constructor using a string is deprecated in this version (or at least should be).
Try to also fix other examples if necessary.

Reply:
Hi @mmmarinho, Indeed, the use of the string related to the convention in DQ_SerialManipulatorDH and DQ_SerialManipulatorMDH is deprecated in [this version](https://github.com/dqrobotics/matlab/pull/69). 
Specifically, there are  warnings about it in both (DQ_SerialManipulatorDH and DQ_SerialManipulatorMDH):
`warning('DQ_SerialManipulatorDH(A,convention) is deprecated. Please use DQ_SerialManipulatorDH(A) instead.');`

and

`warning('DQ_SerialManipulatorMDH(A,convention) is deprecated. Please use DQ_SerialManipulatorMDH(A) instead.');`,

respectively. 

I did the required changes. I was wondering if a warning is enough for deprecation, or the class must raise an error directly.

Best regards, 

Juancho

Reply:
Hi @juanjqo, 

We need to ensure backward compatibility for a while to ensure a smooth transition to newer versions of DQ Robotics. Therefore, our current policy is to raise a warning for deprecated methods in the transition period, and then after the transition period, it becomes an error.
 
Kind regards,
Bruno